05/components/Menu.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import MenuItems from './MenuItems'
+import PropTypes from 'prop-types'
+
+class Menu extends React.Component {
+    render() {
+        const items = this.props.items
+
+        return <ul>
+            {items.map(
+                (item, index) => { return <MenuItems url={item.url} text={item.text} _key={index.toString()}></MenuItems> }
+            )}
+        </ul>
+    }
+}
```

Comment:
Cały ten element mógłby być `<MenuItems>`, który zawiera `<MenuItem>`.
Ponieważ są to podobne nazwy to ja bym zostawił `<Menu>` lub `<MenuList>`